releng/MWE.setup

```diff
@@ -10,7 +10,7 @@
     xmlns:setup.targlets="http://www.eclipse.org/oomph/setup/targlets/1.0"
     xmlns:setup.workingsets="http://www.eclipse.org/oomph/setup/workingsets/1.0"
```

Comment:
mwe.setup is not maintained for years. have no idea what the current state is

Reply:
Didn't look too bad, but I can check it in detail.